service/awsconfig/v10/version_bundle.go

```diff
@@ -19,6 +19,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Masked systemd-networkd-wait-online unit.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "cloudconfig",
+				Description: "Improved encryption key injection.",
+				Kind:        versionbundle.KindChanged,
+			},
```

Comment:
I'd say this is a bugfix for aws-operator. Previous release was passing the key insecurely.

Reply:
Done.